docs/usage/training_time_compression/quantization_aware_training/Usage.md

```diff
@@ -0,0 +1,96 @@
+# Use Neural Network Compression Framework (NNCF) for Quantization Aware Training in PyTorch
+
+This is a step-by-step tutorial on how to integrate the NNCF package into the existing PyTorch project.
+The use case implies that the user already has a training pipeline that reproduces training of the model in the floating  point precision and pretrained model.
+The task is to prepare this model for accelerated inference by simulating the compression at train time.
+Please refer to this [document](/docs/usage/training_time_compression/other_algorithms/LegacyQuantization.md) for quantization implementation details.
+
+## Basic usage
+
+### Step 1: Apply Post Training Quantization to the Model
+
+Quantize the model using the [Post Training Quantization](../../post_training_compression/post_training_quantization/Usage.md) method.
+
+```python
+model = TorchModel() # instance of torch.nn.Module
+quantized_model = nncf.quantize(model, ...)
+```
+
+### Step 2: Run the training pipeline
+
+At this point, the NNCF is fully integrated into your training pipeline.
+You can run it as usual and monitor your original model's metrics and/or compression algorithm metrics and balance model metrics quality vs. level of compression.
+
+Important points you should consider when training your networks with compression algorithms:
```

Comment:
Is it valid? Especially sparsity mentions

Reply:
Sparsity part is invalid, but dropout disabling is a valid point